apps/mobile/src/state/storage-persistors.ts

```diff
@@ -43,9 +88,10 @@ export function mnemonicStore(fingerprint: string) {
       }
       return mnemonicSchema.parse({ mnemonic });
     },
-    async setMnemonic(mnemonic: string, passphrase?: string) {
-      if (passphrase) await SecureStore.setItemAsync(passphraseKey, passphrase, secureStoreConfig);
-      return SecureStore.setItemAsync(fingerprint, mnemonic, secureStoreConfig);
+    async setMnemonic(mnemonic: string, biometrics: boolean, passphrase?: string) {
```

Comment:
Passing in booleans can make reading the [fn signatures unclear](https://github.com/ryanmcdermott/clean-code-javascript?tab=readme-ov-file#dont-use-flags-as-function-parameters)

`setMnemonic(mnemonic, passphrase, { biometrics: boolean }` is much more readable

Why is it we need it configurable again?